Cargo.toml

```diff
@@ -17,6 +17,7 @@ env_logger = "0.8"
 envsubst = "0.2"
 fail = "0.4"
 failure = "0.1"
+failure_derive = "0.1.8"
```

Comment:
Let's do https://github.com/coreos/zincati/issues/519 instead of going this way.